3p/integration.js

```diff
@@ -152,9 +152,18 @@ const AMP_EMBED_ALLOWED = {
   zergnet: true,
 };
 
-const data = parseFragment(location.hash);
-window.context = data._context;
-
+const iframeName = window.name;
```

Comment:
i actually meant to add a comment why we cache this value.
Otherwise, you might want to move master selection code below all the code that read window.name.